stdlib/src/shutil/shutil.mojo

```diff
@@ -0,0 +1,74 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+"""Implements a number of high-level operations on files and
+collections of files.
+
+You can import these APIs from the `shutil` package. For example:
+
+```mojo
+from shutil import rmtree
+```
+"""
+
+from os import listdir
+from os.path import islink, isdir, isfile
+
+
+fn rmtree(path: String, ignore_errors: Bool = False) raises:
```

Comment:
@JoeLoser @ConnorGray Do you have any opinions if we should make `ignore_errors` keyword-only? Python doesn't do this, but it's not clear if that's just a result of backward compatibility. Otherwise, I always feel a bit strange about flags that can be specified positionally.

Reply:
Making `ignore_errors` keyword-only SGTM.